src/Browscap.php

```diff
@@ -44,8 +52,8 @@
     /**
      * Browscap constructor.
      *
-     * @param \Psr\SimpleCache\CacheInterface  $cache
-     * @param LoggerInterface $logger
+     * @param \Psr\SimpleCache\CacheInterface $cache
+     * @param LoggerInterface                 $logger
```

Comment:
Please let's avoid adding space unnecessarily (no need to align arguments like this, just creates noisy diffs)

Reply:
fixed